### ISSUE TITLE
Feature Request: Foxy Gestures mousegesture support.

### ISSUE THREAD

**Issue by somokol** [stoically/firefox-add-on-temporary-containers#50]
I left a request on Firefox's addon site for FoxyGesture support, which you kindly responded to inquire about API support from the FoxyGetures Dev.

I contacted the FG dev on their [Git support page](https://github.com/marklieberman/foxygestures/issues/223) and they responded with the following

> @somokol Can you bring me into your original conversation with the dev of temp containers? I don't think its necessary to have a whole system to register events. I think all that is necessary is for temp containers to add a listener using runtime.onMessageExternal. Then FG can invoke temp containers commands via a trivial user script.

Just thought I'd post that up here to see if this is feasible.

**Comment by stoically** [comment]
As mentioned in the FG Issue it's enough to implement a message-handler and write a small FG UserScript, no API needed. I'll look into it ASAP.

**Comment by stoically** [comment]
Just published version 0.39 with support for other WebExtensions. UserScripts that you could use in FoxyGestures are:

Open New Temporary Container Tab
```js
const url = data.element && data.element.linkHref;
if (url) {
    browser.runtime.sendMessage('{c607c8df-14a7-4f28-894f-29e8722976af}', {
        method: 'createTabInTempContainer'
    });
}
```

Open Link in New Temporary Container Tab
```js
const url = data.element && data.element.linkHref;
if (url) {
    browser.runtime.sendMessage('{c607c8df-14a7-4f28-894f-29e8722976af}', {
        method: 'createTabInTempContainer',
        active: true,
        url
    });
}
```

Open Link in New Background Temporary Container Tab
```js
const url = data.element && data.element.linkHref;
if (url) {
    browser.runtime.sendMessage('{c607c8df-14a7-4f28-894f-29e8722976af}', {
        method: 'createTabInTempContainer',
        active: false,
        url
    });
}
```

Hope you like it. Let me know if something doesn't work as expected.

**Comment by somokol** [comment]
Wow! Thank You! So far, so good. It's working great on my FF 58.0.2.

Thanks again. This is such a perfect addition, to a perfect addon.
